dali/pipeline/operators/operator.h

```diff
@@ -200,6 +212,14 @@ class Operator : public OperatorBase {
    */
   virtual void RunImpl(Workspace<Backend> *ws, int idx = 0) = 0;
 
+  int SequenceSize(int idx = 0) {
+    if (!sequences_allowed_) {
+      return 1;
```

Comment:
sequence size 1 implies that there is a sequence. How about returning something else in case that sequences are not allowed?

Reply:
Done